cluster/node-pools/master-default/userdata.yaml

```diff
@@ -496,10 +483,6 @@ write_files:
               -> disableAccessLog()
               -> setPath("/metrics")
               -> "https://127.0.0.1:9085";
-            image_policy_webhook: Path("/image-policy-webhook")
```

Comment:
This is referenced in the Prometheus configuration, could you update that as well?

Reply:
Thanks!